README.md

```diff
@@ -55,6 +87,13 @@ class BasicDataProvider extends DataProvider {
 A basic Importer
 ----------------
 
+Every data importer must extend the ``AbstractImporter`` abstract class or implement the interface ```ImporterInterface```.
+
+In the `processRecord` method you handle the processing of every record, such as creating Content Repository node for each incoming data record.
+ 
+Do not forget to register the processed nodes with `registerNodeProcessing`. The method will handle feature like logging and tracking of imported node to decide
```

Comment:
The method will handle features like logging and the tracking of imported nodes to decide